src/lib/fetch-handler.js

```diff
@@ -3,6 +3,7 @@ const ResponseBuilder = require('./response-builder');
 const FetchMock = {};
 
 FetchMock.fetchHandler = function (url, opts) {
+	url = url instanceof Request ? url.url : url;
```

Comment:
rather than `Request` use `this.config.Request` as this is resilient to people using custom `fetch` implementations